changelog/issue-5514.md

```diff
@@ -0,0 +1,7 @@
+audience: deployers
+level: minor
```

Comment:
maybe major since it will break local db?

Reply:
I'm not sure.. I'm tempted to keep it as minor since this PR still allows for v11 to be running. It only breaks the local db if they migrate their db to v15. Wdyt?

Reply:
sure, i'm not insisting on major. But adding a note here in changelog would be good 👍